src/js/DrawStyleController.js

```diff
@@ -179,6 +198,10 @@ goog.require('ga_styles_service');
         {id: 'telephone'},
         {id: 'waste-basket'},
         {id: 'water'}
+      ],
+      iconCategories: [
+        {label: 'standard', colorOption: true, nrIcons: 114},
```

Comment:
Adds an `id` property,  a label can't change over the time.  
`colorOption` is not explicit, use `useColorOption` or `hasColorOption`, with that you know the variable is a boolean. 
Usally we use `nb` as shortcut for number not `nr` 
For `standard` category  the number of icons is not used so don't set it.